OCP/app/delivery_to_stdout.rb

```diff
@@ -0,0 +1,11 @@
+class DeliveryToStdout
+  def initialize(overrides = {})
+    @logger_to_stdout = overrides.fetch(:logger_to_stdout) { (STDOUT) }
```

Comment:
```suggestion
    @logger_to_stdout = overrides.fetch(:logger_to_stdout) { STDOUT }
```
Não precida dos `()`